source/includes/read/change-streams.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// start-db-coll
+$collection = $client->sample_restaurants->restaurants;
+// end-db-coll
+
+// Monitors and prints changes to the "restaurants" collection
+// start-open-change-stream
+$changeStream = $collection->watch();
+
+foreach ($changeStream as $event) {
+    echo json_encode($event) . PHP_EOL;
```

Comment:
Since the events are BSON documents, it'd be preferable to use the existing BSON functions to output JSON. Relying on PHP's `json_encode()` functions will not produce Extended JSON for types that need a special representation.

This is why the existing examples in PHPLIB use a common help function:

```php
function toJSON(object $document): string
{
    return MongoDB\BSON\Document::fromPHP($document)->toRelaxedExtendedJSON();
}
```

Reply:
Noted that you added the `toJSON()` utility function to the docs.